src/Resources/config/config.yaml

```diff
@@ -2,3 +2,17 @@ sitemap:
     static_routes:
         - { route: sylius_shop_homepage }
         - { route: sylius_shop_contact_request }
+
+parameters:
+    sylius.sitemap.filesystem: sylius_sitemap
+    sylius.sitemap.path: "%kernel.project_dir%/data/sitemap"
```

Comment:
Should probably be put in `var/storage/sylius-sitemap` or something to keep of with conventions

Reply:
I was a bit in doubt about this, I read so many conflicting things on it and don't want people to flush cache/remove var folders and having to regenerate. Any good read on what's the Symfony default suggestion? I propose to solve this also in another PR, as we are not close to a release yet anyway.